app/router/session_group.py

```diff
@@ -31,6 +31,11 @@ def get_group_for_session(session_id: str):
     response = requests.get(session_group_db_url + session_id)
     if response.status_code == 200:
         if len(response.json()) != 0:
-            return response.json()
+            data = response.json()
+            group_type_response = requests.get(group_type_db_url, params= {'type': 'group', 'child_id': data['group_type_id']})
+            if group_type_response.status_code == 200:
+                if len(group_type_response.json()) != 0:
+                    group_data = group_type_response.json()
+                    return group_data[0]['child_id']
```

Comment:
is there where we'd have to extend if we need to allow multiple groups per session? instead of `group_data[0]` we send `group_data[i]['child_id'] for i in len(group_data)]`